src/Featurizers/Components/FrequencyEstimator.h

```diff
@@ -0,0 +1,169 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include <limits>
+
+#include "TrainingOnlyEstimatorImpl.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+namespace Components {
+
+// TODO: This should be removed in favor of the StatisticalMetricsEstimator

```

Comment:
I don't understand this comment. Is this saying that frequency detection should become part of the StatisticalMetricsEstimator?

Reply:
No. It's a comment I forgot to remove.